src/datasources/userDataCreateUser.datasource.js

```diff
@@ -6,13 +6,16 @@ const { url, ttl, prefix } = config.datasources.userdata;
  */
 export async function load({ smaugUserId }, context) {
   const addUserEndpoint = url + "user/add";
-  await context.fetch(addUserEndpoint, {
+  const res = await context.fetch(addUserEndpoint, {
     headers: {
       "Content-Type": "application/json",
     },
     method: "POST",
     body: JSON.stringify({ smaugUserId: smaugUserId }),
   });
+  if (res?.status !== 200) {
+    throw new Error(res?.body?.error || "Something went wrong");
```

Comment:
måske "could not add user"

Reply:
Den viser en fejlbesked fra userdata servicen (fra res.body.error) som er "Failed to create user", men jeg tilføjer det også istedet for "Somehting went wrong"